packages/build/tests/core/tests.js

```diff
@@ -522,6 +522,21 @@ test.serial('functions can have a config with different parameters passed to zip
   t.snapshot(normalizeOutput(output))
 })
 
+test.serial('internalSrcFolder is passed to zip-it-and-ship-it and helps prefill the generator field', async (t) => {
```

Comment:
We already have a generic test that checks which properties are passed to zip-it-and-ship-it: https://github.com/netlify/build/blob/main/packages/build/tests/core/tests.js#L339. Would it make sense to reuse it?